flightmate/src/main/java/com/flightmate/dao/FlightDao.java

```diff
@@ -0,0 +1,19 @@
+package com.flightmate.dao;
+
+package com.flightmate.dao;
```

Comment:
```suggestion
package com.flightmate.dao;
```
Remove this, because it's already on line 1.